elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidator.java

```diff
@@ -74,18 +84,55 @@ public static void main(String[] args) throws IOException, ParseException, Proce
             throw new MissingOptionException("Missing required option");
         }
         String configDir = cli.getOptionValue("configDir");
-        File file = new File(configDir);
-        String absoluteBasePath = file.getAbsolutePath();
-        log.info("Absolute Path for Model Configs Directory: " + absoluteBasePath);
 
-        if (!file.isDirectory()) {
+        DynamicConfigValidator dynamicConfigValidator = new DynamicConfigValidator();
+
+        if (exists(configDir)) {
+            //look in local file system
+            File configFile = new File(configDir);
+            String absoluteBasePath = configFile.getAbsolutePath();
+            log.info("Absolute Path for Model Configs Directory: " + absoluteBasePath);
+
+            dynamicConfigValidator.readAndValidateConfigs(absoluteBasePath);
+
+        } else {
+            // look in classpath
+            log.info("Looking at " + configDir + " in resources");
+            dynamicConfigValidator.readAndValidateClasspathConfigs(configDir);
+        }
+
+        log.info("Configs Validation Passed!");
+    }
+
+    /**
+     * Read and validate config files under resources.
+     * @throws ProcessingException
+     * @throws IOException
+     */
+    public void readAndValidateClasspathConfigs(String filePath)
+            throws ProcessingException, IOException {
+
+        this.setConfigDir(DynamicConfigHelpers.formatFilePath(formatClassPath(filePath)));
+        ClassLoader classLoader = DynamicConfigValidator.class.getClassLoader();
```

Comment:
This seems like reading from the classpath the hard way.   In all other instances where I've had to read something from the classpath - it was a one-liner.